apps/coscrad-frontend/src/configurable-front-matter/data/content.config.SAMPLE.ts

```diff
@@ -5,8 +5,10 @@ export const contentConfig: ConfigurableContent = {
     siteTitle: 'COSCRAD [~Sandbox~]',
     subTitle: 'Powering a Web of Knowledge',
     about: 'Welcome to the official COSCRAD sandbox!',
+    enableLoginButton: false,
```

Comment:
`shouldEnableAdminMode: true` // use true in the sample because most builds want this value